src/packet/BinaryWriter.js

```diff
@@ -1,137 +1,62 @@
-﻿'use strict';
+'use strict';
 /*
  * Simple BinaryWriter is a minimal tool to write binary stream with unpredictable size.
  * Useful for binary serialization.
  *
- * Copyright (c) 2016 Barbosik
- * 
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- *     http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 
- * 
+ * Copyright (c) 2016 Barbosik https://github.com/Barbosik
+ * License: Apache License, Version 2.0
  */
 
-function BinaryWriter(size) {
-    if (!size || size <= 0) {
-        size = Buffer.poolSize / 2;
-    }
-    this._buffer = new Buffer(size);
+function BinaryWriter() {
+    this._buffer = parseInt(process.version[1]) < 6 ? new Buffer(1e5) : Buffer.allocUnsafe(1e5);

```

Comment:
@Andrews54757
> MultiOgar is unreliable with this change. Try to use the updates branch as a public server for a week.
- If you insist so. I'll surely forget for this.

> Also, one of multiOgar-edited's goals is to be faster than the original multiOgar.
- I don't know what Megabyte918 was thinking for the fork. I think it has the same performance as MultiOgar. Mine is around 9% slower than MultiOgar-Edited.

> V8 is dumb enough. You have no idea.
- See [this article](http://v8project.blogspot.rs/2015/08/getting-garbage-collection-for-free.html). Even though it explains it with rendering sessions there aren't many differences with a game server's situation.
- The buffers always die young.
- And poof. Buffers become unallocated.

> I do underestimate you. You talk about things that you dont even understand.
- So you still think you know me? Answer these questions then.
- Where do I live? (Actually an easy one)
- What is my real name?
- Which languages do I know, both real world and programming languages?
- When was I born?
- What is my favorite color?
- When did I start programming?
- What was my first programming language?
- What was the first game I played a good amount of time? When did I play it?
- When did I start programming in Javascript?
- When have I made my first edit in Javascript? When was it?
- What's the name of my reddit account?
- What's the name of my old reddit account?
- Do I have a YouTube account?
- When did I experiment with child processes? In what ways did I experiment with them?
- Am I involved in any other things, besides Ogar?